proto/src/prost/chainmain.nft.v1.rs

```diff
@@ -108,292 +111,321 @@ pub mod msg_client {
-                )
-            })?;
+            self.inner
+                .ready()
+                .await
+                .map_err(|e| {
+                    tonic::Status::new(
+                        tonic::Code::Unknown,
+                        format!("Service was not ready: {}", e.into()),
+                    )
+                })?;
             let codec = tonic::codec::ProstCodec::default();
-            let path = http::uri::PathAndQuery::from_static("/chainmain.nft.v1.Msg/TransferNFT");
+            let path = http::uri::PathAndQuery::from_static(
+                "/chainmain.nft.v1.Msg/TransferNFT",
+            );
             self.inner.unary(request.into_request(), path, codec).await
         }
-        #[doc = " BurnNFT defines a method for burning a nft."]
+        /// BurnNFT defines a method for burning a nft.
         pub async fn burn_nft(
             &mut self,
             request: impl tonic::IntoRequest<super::MsgBurnNft>,
         ) -> Result<tonic::Response<super::MsgBurnNftResponse>, tonic::Status> {
-            self.inner.ready().await.map_err(|e| {
-                tonic::Status::new(
-                    tonic::Code::Unknown,
-                    format!("Service was not ready: {}", e.into()),
-                )
-            })?;
+            self.inner
+                .ready()
+                .await
+                .map_err(|e| {
+                    tonic::Status::new(
+                        tonic::Code::Unknown,
+                        format!("Service was not ready: {}", e.into()),
+                    )
+                })?;
             let codec = tonic::codec::ProstCodec::default();
-            let path = http::uri::PathAndQuery::from_static("/chainmain.nft.v1.Msg/BurnNFT");
+            let path = http::uri::PathAndQuery::from_static(
+                "/chainmain.nft.v1.Msg/BurnNFT",
+            );
             self.inner.unary(request.into_request(), path, codec).await
         }
     }
 }
 /// BaseNFT defines a non-fungible token
-#[derive(Serialize, Deserialize, Clone, PartialEq, ::prost::Message)]
+#[derive(Serialize, Deserialize)]
+#[derive(Clone, PartialEq, ::prost::Message)]
 pub struct BaseNft {
-    #[prost(string, tag = "1")]
+    #[prost(string, tag="1")]
     pub id: ::prost::alloc::string::String,
-    #[prost(string, tag = "2")]
+    #[prost(string, tag="2")]
     pub name: ::prost::alloc::string::String,
-    #[prost(string, tag = "3")]
+    #[prost(string, tag="3")]
     pub uri: ::prost::alloc::string::String,
-    #[prost(string, tag = "4")]
+    #[prost(string, tag="4")]
     pub data: ::prost::alloc::string::String,
-    #[prost(string, tag = "5")]
+    #[prost(string, tag="5")]
     pub owner: ::prost::alloc::string::String,
 }
 /// Denom defines a type of NFT
-#[derive(Serialize, Deserialize, Clone, PartialEq, ::prost::Message)]
+#[derive(Serialize, Deserialize)]
+#[derive(Clone, PartialEq, ::prost::Message)]
```

Comment:
Is it because no formating?

Reply:
that's the output I got after running `make proto`

Reply:
there's formatting using https://github.com/dtolnay/prettyplease

Reply:
> This library is designed using the simplest possible algorithm and data structures that can deliver about 95% of the quality of rustfmt-formatted output.